### PR TITLE
[14.0][MIG] purchase_request_to_requisition

### PR DESCRIPTION
**Pull request by kittiu**
Migration from v9 -> v14

### CODE REVIEW

**Comment by newtratip** [comment]
@kittiu, License should be AGPL ?

**Comment by kittiu** [comment]
> @kittiu, License should be AGPL ?

Done, thanks.

**Comment by kittiu** [comment]
Runbot these days seem to be red even success test.
```
+=======================================
|  Tests summary:
|---------------------------------------
| test_server.py              [1;32mSuccess[0;m
+=======================================
 * Stopping PostgreSQL 9.6 database server       [80G 
[74G[ OK ]
```

File: purchase_request_to_requisition/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
```
Comment (newtratip) [inline purchase_request_to_requisition/__manifest__.py:2]: Change all *.py files or in xml files (if any)

```suggestion
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
```

File: purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition_view.xml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                        <button
+                            name="make_purchase_requisition"
+                            type="object"
+                            class="btn-primary"
+                            string="Create Purchase Agreements"
+                        />
+                        <button special="cancel" string="Cancel" class="oe_link" />
+                    </footer>
+<!--
+                     <group colspan="2">
```
Comment (newtratip) [inline purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition_view.xml]: Please delete comment lines

File: purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition_view.xml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                name="view_id"
+                ref="view_purchase_request_line_make_purchase_requisition"
+            />
+            <field name="target">new</field>
+            <field
+                name="binding_model_id"
+                ref="purchase_request.model_purchase_request_line"
+            />
+        </record>
+
```
Comment (newtratip) [inline purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition_view.xml:72]:
```suggestion
```

File: purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+    _name = "purchase.request.line.make.purchase.requisition"
+    _description = "Purchase Request Line Make Purchase Agreement"
+
+    item_ids = fields.One2many(
+        "purchase.request.line.make.purchase.requisition.item", "wiz_id", string="Items"
+    )
+    purchase_requisition_id = fields.Many2one(
+        "purchase.requisition",
+        string="Purchase Agreement",
+        required=False,
```
Comment (newtratip) [inline purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py:17]: by default `required=False`, No need this
```suggestion
        
```

File: purchase_request_to_requisition/models/purchase_request.py [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+        ._fields["state"]
+        .selection,
+        store=True,
+    )
+    is_editable = fields.Boolean(compute="_compute_is_editable", string="Is editable")
+
+    def unlink(self):
+        for line in self:
+            if line.requisition_lines:
+                raise exceptions.Warning(
```
Comment (newtratip) [inline purchase_request_to_requisition/models/purchase_request.py:121]:
```suggestion
                raise exceptions.UserError(
```

**Comment by bruno-zanotti** [comment]
Hello @kittiu @newtratip 

I am interested in migrating this module to 13.0, actually, I already made a draft PR (https://github.com/OCA/purchase-workflow/pull/1315) base on this one (thanks for that). Is there something remaining to do o just waiting for OCA review?

**Comment by kittiu** [comment]
> Hello @kittiu @newtratip
> 
> I am interested in migrating this module to 13.0, actually, I already made a draft PR (#1315) base on this one (thanks for that). Is there something remaining to do o just waiting for OCA review?

We are using it in production now. Yes, only pending review.

**Comment by kittiu** [comment]
@Bruno-Zanotti can you help review this? As you proceed with v13, may be you have good suggestion.

**Review by Saran440 — APPROVED** [review]
Functional Test :+1:

**Review by newtratip — APPROVED** [review]
LGTM :+1:

**Pull request merged**